src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/Sdk/AutoImport.props

```diff
@@ -16,7 +16,7 @@ https://github.com/dotnet/designs/blob/4703666296f5e59964961464c25807c727282cae/
 -->
 <Project>
 
-  <ItemGroup Condition=" '$(TargetPlatformIdentifier)' == 'android' and $([MSBuild]::VersionEquals($(TargetFrameworkVersion), '8.0')) and ('$(ImplicitUsings)' == 'true' or '$(ImplicitUsings)' == 'enable') ">
+  <ItemGroup Condition=" '$(TargetPlatformIdentifier)' == 'android' and 'AndroidDisableImplicitUsings' != 'true' and $([MSBuild]::VersionEquals($(TargetFrameworkVersion), '8.0')) and ('$(ImplicitUsings)' == 'true' or '$(ImplicitUsings)' == 'enable') ">
```

Comment:
Probably should rename this to `XamarinDisableImplicitUsings` and use it [here](https://github.com/xamarin/xamarin-macios/blob/1fd05057ed96083fc70ba0f0afb8bb544592ad3b/dotnet/targets/Microsoft.Sdk.ImplicitNamespaceImports.template.props#L25) as well?

Reply:
https://github.com/xamarin/xamarin-macios/pull/18017